aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1370,11 +1370,20 @@ Tensor narrow_symint(const Tensor& self, int64_t dim, SymInt start, SymInt lengt
 
 // This overload exists purely for XLA, because they wanted to pass in "symbolic"
 // start via Tensor.
-Tensor narrow_tensor_symint(const Tensor& self, int64_t dim, const Tensor& start, SymInt length) {
+Tensor narrow_tensor_symint(const Tensor& grad, IntArrayRef input_sizes, int64_t dim, const Tensor& start, SymInt length) {
```

Comment:
I guess changing this function is a typo?

Reply:
I updated `narrow_tensor_symint` in [aten/src/ATen/native/native_functions.yaml](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml) to `CompositeExplicitAutograd` according to [this review](https://github.com/pytorch/pytorch/pull/99757#discussion_r1180745927) so the function should be updated as well right?